GNUmakefile

```diff
@@ -1000,14 +1000,18 @@ endif
 tools:
 	cd internal/tools && go generate -tags tools ./
 
+LINTFILESCMD=find src/os/ src/reflect/ -type f -name '*.go'
```

Comment:
How about changing this variable to contain a list of directories to scan? That would be easier to maintain/adjust/expand in future.

Reply:
@b0ch3nski Similar to this? https://github.com/tinygo-org/tinygo/pull/4679/commits/7190e019ae8a6715628125025906f037405f6839

Reply:
@cibomahto LGTM :+1: